### PR TITLE
PQ-1  Module Setup - Blank Module

### PR DESCRIPTION
**Pull request by slubwama**
https://issues.openmrs.org/projects/PQ/issues/PQ-1?filter=allopenissues
This is a module setup for patient queueing with a blank module.

### CODE REVIEW

**Comment by HerbertYiga** [comment]
you could go on to squash your commits here!!

**Comment by slubwama** [comment]
> you could go on to squash your commits here!!

@HerbertYiga  Done

**Comment by slubwama** [comment]
@HerbertYiga  @dkayiwa any progress on this PR

File: api/src/main/java/org/openmrs/module/patientqueueing/api/dao/PatientQueueingDao.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.api.db.hibernate.DbSession;
+import org.openmrs.api.db.hibernate.DbSessionFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository("patientqueueing.PatientQueueingDao")
+public class PatientQueueingDao {
+	
+	protected final Log log = LogFactory.getLog(getClass());
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/module/patientqueueing/api/dao/PatientQueueingDao.java:22]: Did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-loggingLogging

**Pull request merged**